aws/resource_aws_cloudformation_stack.go

```diff
@@ -648,3 +648,77 @@ func cfStackEventIsStackDeletion(event *cloudformation.StackEvent) bool {
 		*event.ResourceType == "AWS::CloudFormation::Stack" &&
 		event.ResourceStatusReason != nil
 }
+
```

Comment:
Add these functions for use in the new S3 bucket acceptance tests. Future opportunity to use these in the `aws_cloudformation_stack` resource functionality.